index.md

```diff
@@ -106,8 +106,8 @@ Videoconferencing software that is reliable and offers high quality audio and vi
 
 | Tools we've tried                             | Cost                                        | Ease of use                                                                                          | Remarks and concerns    |
 | -------------------------------- | ------------------------------------------- | --------------------------------------------------------------------------------------------------- | --- |
-| [Jitsi](https://meet.jit.si/) 🍄 | Free                                        | Runs in the browser without needing to download another app. Hypha hosts our own Jitsi server.  | Audio and video quality is inferior to Zoom. Browser-based client can present connection issues, particularly if using Firefox. |
-| [BigBlueButton](https://bigbluebutton.org/) 🍄 | Free                           | Runs in the browser. Users can upload presentations to present. Collaborative tools like whiteboards, laser pointer, participant emoji/mood. |   Browser-based client can present connectivity issues. No official public servers.  |
+| [Jitsi](https://meet.jit.si/) 🍄 | Free                                        | Runs in the browser without needing to download another app. Hypha hosts our own Jitsi server.  | Audio and video quality is inferior to Zoom. Browser-based client can present connection issues, particularly if using Firefox. Audio-only on Safari, and does not work reliably. Mobile applications work well on Android and iOS. |
```

Comment:
unclear if you mean Safari doesn't work reliably in second sentence.

Reply:
Thanks! Fixing the sentence.

```suggestion
| [Jitsi](https://meet.jit.si/) 🍄 | Free                                        | Runs in the browser without needing to download another app. Hypha hosts our own Jitsi server.  | Audio and video quality is inferior to Zoom. Browser-based client can present connection issues, particularly if using Firefox. On Safari, there is no video support and the audio does not work reliably. Mobile applications work well on Android and iOS. |
```